src/__fixtures__/mockGlobalSettings.ts

```diff
@@ -1,4 +1,4 @@
-import { GlobalSettings } from '../../types';
+import { GlobalSettings } from '../types';
 
 export const mockGlobalSettings: GlobalSettings = {
     globalComponents: {
```

Comment:
A bit of a nitpick (and purely opinion), but since this is not exactly source code, what do you think of moving it one level further up, out of the `src` directory?

Reply:
I had it there originally, but moved it. I can totally move it back.